src/main/java/knight/ToDo.java

```diff
@@ -0,0 +1,20 @@
+package knight;
+
+/**
+ * Represents a task that needs to be done.
+ */
+public class ToDo extends Task {
```

Comment:
I believe ToDo should be named as `Todo`? (According to the cs2103 convention that used the example of `Id` and not `ID`).